VerilogCodingStyle.md

```diff
@@ -1527,6 +1526,33 @@ Do not use positional arguments to connect signals to ports.
 
 Instantiate ports in the same order as they are defined in the module.
 
+Align expressions in [tabular style](#tabular-alignment).
```

Comment:
Maybe "port expressions" rather than just "expressions"? I don't have a strong view.

Reply:
The SystemVerilog spec calls it "expression", but I also find it too generic. I updated the text to read "port expressions", as suggested.